docs/src/main/docbook/client.xml

```diff
@@ -1130,4 +1130,61 @@ Client client = ClientBuilder.newBuilder().sslContext(sslContext).build();</prog
             }
         </programlisting>
     </section>
+    <section>
+        <title>Header Expect:100-continue support</title>
+        <para>
+        This section describes support of Expect:100-continue in Jersey client using Expect100Continue feature.
+        Jersey client supports given header for default JDK HTTP connector only.
+        </para>
+        <para>
+        <emphasis>Jersey client Expect100Continue feature</emphasis>
+        </para>
+        <para>
+            Since Jersey 2.32 it is possible to send Expect:100-continue header from Jersey client. Feature shall be
+            registered in client using (for example)
+            <programlisting language="java" linenumbering="numbered">
+                target(RESOURCE_PATH).register(Expect100ContinueFeature.basic());
+            </programlisting>
+            Note that registration can be done in several ways: with basic settings, and with custom settings:
+            <programlisting language="java" linenumbering="numbered">
+                target(RESOURCE_PATH).register(Expect100ContinueFeature.withCustomThreshold(100L));
+            </programlisting>
+            Basic registration means that default sending threshold will be used. Value of the default threshold is
+            <programlisting language="java" linenumbering="numbered">
+                DEFAULT_EXPECT_100_CONTINUE_THRESHOLD_SIZE = 65536L;
+            </programlisting>
+            Threshold is used to determine allowed size of request after which 100-continue header shall be sent before
+            sending request itself.
+        </para>
+        <para>
+            <emphasis>Environment properties configuration</emphasis>
+        </para>
+        <para>
+            Previous paragraph described programmatic way of configuration. However the Expect100Continue feature can
+            be configured using environment variables as well.
+        </para>
+        <para>
+            Since Jersey client can be influenced through environment variables, there are two variables which come
+            since Jersey 2.32:
+            <programlisting language="bash" linenumbering="unnumbered">
+                -Djersey.config.client.request.expect.100.continue.processing=true/false
```

Comment:
Make a note that the -Djersey.config.allowSystemPropertiesProvider=true property needs to be set, too.